### DISCUSSION TITLE
is node affinity supported by strimzi operator pod?

### DISCUSSION THREAD

**Discussion by saloni-2404** [strimzi/strimzi-kafka-operator#4437] [Q&A] [answered]
"error: error validating "/tmp/operator/deployment-operator.yaml": error validating data: ValidationError(Deployment.spec.template): unknown field "affinity" in io.k8s.api.core.v1.PodTemplateSpec; if you choose to ignore these errors, turn validation off with --validate=false
"

(Providing the values in deployment-operator.yaml)

**Comment by scholzj** [comment, marked as answer]
The cluster-operator deployment is just deployment as any other. You can configure there whatever you want. From the error, I guess you have some typos there or you placed it in wrong place or something (I would expect it in `Deployment.spec.template.spec` for example).

**Reply by saloni-2404** [reply]
Okay, so for kafka, zookeeper, entity-operator, kafka-exporter i tried with template.pod.affinity..
for kafka and zookeeper, it worked, is it expected elsewhere for the others(entity op/kafka exporter)?

**Reply by scholzj** [reply]
I think it works everywhere.
